src/settings/MappingProfiles/detailsSections/view/InstanceDetailsSection/DescriptiveData.js

```diff
@@ -0,0 +1,295 @@
+  const physicalDescriptionsData = transformSubfieldsData(physicalDescriptions, physicalDescriptionsVisibleColumns);
+
+  const natureOfContentTermsVisibleColumns = ['natureOfContentTermId'];
+  const natureOfContentTermsMapping = {
+    natureOfContentTermId: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.instance.descriptiveData.field.natureOfContentTermId`} />
+    ),
+  };
+  const natureOfContentTermsFormatter = { natureOfContentTermId: x => x?.natureOfContentTermId || <NoValue /> };
+  const natureOfContentTermsData = transformSubfieldsData(natureOfContentTerms, natureOfContentTermsVisibleColumns);
+
+  const formatsVisibleColumns = ['instanceFormatId'];
+  const formatsMapping = {
+    instanceFormatId: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.instance.descriptiveData.field.instanceFormatId`} />
+    ),
+  };
+  const formatsFormatter = { instanceFormatId: x => x?.instanceFormatId || <ProhibitionIcon /> };
+  const formatsData = transformSubfieldsData(formats, formatsVisibleColumns);
+
+  const languagesVisibleColumns = ['languageId'];
+  const languagesMapping = {
+    languageId: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.instance.descriptiveData.field.languageId`} />
+    ),
+  };
+  const languagesFormatter = { languageId: x => x?.languageId || <ProhibitionIcon /> };
+  const languagesData = transformSubfieldsData(languages, languagesVisibleColumns);
+
+  const publicationFrequenciesVisibleColumns = ['publicationFrequency'];
+  const publicationFrequenciesMapping = {
+    publicationFrequency: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.instance.descriptiveData.field.publicationFrequency`} />
+    ),
+  };
+  const publicationFrequenciesFormatter = { publicationFrequency: x => x?.publicationFrequency || <ProhibitionIcon /> };
+  const publicationFrequenciesData = transformSubfieldsData(publicationFrequencies, publicationFrequenciesVisibleColumns);
+
+  const publicationRangeVisibleColumns = ['publicationRange'];
+  const publicationRangeMapping = {
+    publicationRange: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.instance.descriptiveData.field.publicationRange`} />
+    ),
+  };
+  const publicationRangeFormatter = { publicationRange: x => x?.publicationRange || <ProhibitionIcon /> };
+  const publicationRangeData = transformSubfieldsData(publicationRange, publicationRangeVisibleColumns);
+
+  return (
+    <Accordion
+      id="descriptive-data"
+      label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.instance.descriptiveData.section`} />}
+    >
+      <Row left="xs">
+        <Col
+          data-test-publications
+          xs={12}
+          className={css.colWithTable}
+        >
+          <div>
+            <strong>
+              <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.instance.descriptiveData.field.publications.legend`} />
+            </strong>
```

Comment:
Why you cover it with `<strong>` tag and don't use css? You use it many times in this file. It will be easier if you add just a class to parents `<div>`